docs/content/v0.20/architecture.md

```diff
@@ -41,7 +41,7 @@ Examples of users interaction with KubeStellar are illustrated in the
 
 The KubeStellar architecture has these main modules:
```

Comment:
The transport controller does not appear in this list. I think it is a big enough deal that it should.

Reply:
FYI: (this PR did not introduce this problem, but makes it worse) it would be very helpful if this list made it clear what comes from the KubeStellar org vs is imported from OCM.